src/main/java/duke/Main.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.ui.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private final Duke duke = new Duke();
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
It seems you have yet to add JavaDoc comments for the methods in this class as you are still working on it. Do remember to add them in when fine tuning your code! 😸